deployment/arm/cosmosdb.json

```diff
@@ -0,0 +1,85 @@
+{
```

Comment:
Thoughts on moving this cosmos resource into the same global template with the other global resources?